testing/criu/APKBUILD

```diff
@@ -1,18 +1,18 @@
 # Contributor: Stuart Cardall <developer@it-offshore.co.uk>
 # Maintainer: Stuart Cardall <developer@it-offshore.co.uk>
 pkgname=criu
-pkgver=3.8
+pkgver=3.8.1
 pkgrel=0
 pkgdesc="Process tree checkpoint/restore utility"
 url="http://criu.org"
 arch="x86_64"
 license="GPL-2.0-or-later LGPL-2.1-only"
-makedepends="xmlto asciidoc python protobuf-c-dev protobuf-dev libaio-dev libnl3-dev
-	py-protobuf libcap-dev perl libnet-dev"
+makedepends="asciidoc libaio-dev libcap-dev libnet-dev libnl3-dev
+	perl protobuf-c-dev protobuf-dev py-protobuf python xmlto"
 checkdepends="py-yaml"
 options="!check" # https://github.com/checkpoint-restore/criu/issues/434
 subpackages="$pkgname-dev $pkgname-doc py-$pkgname:_py:noarch"
-source="$pkgname-$pkgver.tar.gz::https://github.com/xemul/$pkgname/archive/v$pkgver.tar.gz"
+source="$pkgname-$pkgver.tar.gz::https://github.com/xemul/criu/archive/v$pkgver.tar.gz"
```

Comment:
any reason to change this line?

Reply:
I assume it is easier to copy-paste from source. Please, look at this related [comment](https://github.com/alpinelinux/aports/pull/3648) by @ncopa.

Reply:
We definitely need a **best practices** document detailing such details.
- Should I use $pkgname on source="..." or the literal package name?
- Should I sed or should I patch?
and all other points of confusion